application/decoder/cdl.py

```
@@ -0,0 +1,51 @@
+            Parse the log line and save the relevant data.
+        '''
+        delimiter = line[0]
+
+        if delimiter == LINE_TYPE_DELIMITER["IR_HEADER"]:
+            self.header = json.loads(line)
+        elif delimiter == LINE_TYPE_DELIMITER["EXCEPTION"]:
+            self.exception = line
+        elif delimiter == LINE_TYPE_DELIMITER["VARIABLE"]:
+            pass
+        elif delimiter == LINE_TYPE_DELIMITER["UNIQUE_ID"]:
+            pass
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)